src/content/docs/apm/agents/ruby-agent/configuration/distributed-tracing-ruby-agent.mdx

```diff
@@ -0,0 +1,354 @@
+
+For more help finding your traces in the UI:
+
+* [Understand and use the distributed tracing UI](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui)
+* [Query distributed trace data](/docs/understand-dependencies/distributed-tracing/ui-data/query-distributed-trace-data)
+
+## Set up Infinite Tracing (advanced option) [#infinite-tracing]
+
+Standard distributed tracing for APM agents ([above](#quick-start-apm)) captures up to 10% of your traces, but if you want us to analyze all your data and find the most relevant traces, you can set up Infinite Tracing.
+
+<Callout variant="tip">
+  To learn more about this feature, see [Infinite Tracing](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing).
+</Callout>
+
+Before beginning, first ensure you meet [the requirements](/docs/understand-dependencies/distributed-tracing/infinite-tracing/introduction-infinite-tracing#requirements).
+
+### Step 1. Complete the new agent installation [#install-agents-first]
+
+The Infinite Tracing setup builds on the instrumentation step from the [new agent installation](#quick-start-apm) for standard distributed tracing. After you finish installing the agents, continue with the trace observer setup.
+
+### Step 2. Set up the trace observer [#provision-trace-observer]
+
+The trace observer is a New Relic AWS-based service that collects and analyzes all your traces. Follow the instructions in [Set up trace observer](/docs/understand-dependencies/distributed-tracing/infinite-tracing/set-trace-observer). When you're done, return here with your trace observer information and continue with the next step to configure the agent.
+
+### Step 3: Configure the agent for Infinite Tracing
+
+Infinite Tracing configuration settings include the standard distributed tracing plus information about the trace observer. Note that server-side configuration is not available for Infinite Tracing.
+
+<CollapserGroup>
+  <Collapser
+    id="ruby-config"
+    title="Ruby Infinite Tracing configuration "
+  >
+    Here's an overview of the settings. For more help with configuration, see [Ruby agent configuration](/docs/agents/ruby-agent/configuration/ruby-agent-configuration#distributed_tracing).
+
+    To set up Infinite Tracing, you need to install the Infinite Tracing gem. The gem is available in [rubygems.org](https://rubygems.org/gems/newrelic-infinite_tracing). For applications using Bundler, additionally include the Infinite Tracing gem in the Gemfile:
+
+    ```
+    gem 'newrelic-infinite_tracing'
+    ```
+
+    If you're using Rails 3 or higher, or Rails 2.3 in the [recommended configuration](http://bundler.io/v1.12/rails3.html), Rails will automatically call `Bundler.require` and cause `newrelic-infinite_tracing` to be required during startup of your application. If you're using Sinatra or another framework, you must manually call `require 'newrelic/infinite_tracing'` or manually call `Bundler.require`.
```

Comment:
Rails 2.3 is no longer supported and Rails 3.x (now 12 years old) itself is deprecated. I recommend leaving out the reference to Rails 2 and the related Bundler documentation:

```diff
- If you're using Rails 3 or higher, or Rails 2.3 in the [recommended configuration](http://bundler.io/v1.12/rails3.html), Rails
+ If you're using Ruby on Rails version 3 or higher, Rails
```